# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/RangeUtils.cpp
# Commit: 82ee04923c59
# Full Hash: 82ee04923c5942496fba9cc19feab47e54f5926f
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-05-09 04:15:00
# Regressor Bug: 1932150
# File Overlap Count: 1
# Description:
#   Bug 1932150 - Update ranges to correctly set and update nodes for flattened selection r=jjaschke,smaug,dom-core
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231589
# ==============================================================================

diff -r a4ef62ad447f -r 82ee04923c59 dom/base/RangeUtils.cpp
--- a/dom/base/RangeUtils.cpp	Thu May 08 13:37:20 2025 +0000
+++ b/dom/base/RangeUtils.cpp	Thu May 08 13:37:21 2025 +0000
@@ -10,6 +10,7 @@
 #include "mozilla/dom/AbstractRange.h"
 #include "mozilla/dom/Document.h"
 #include "mozilla/dom/ShadowRoot.h"
+#include "mozilla/dom/HTMLSlotElement.h"
 #include "nsContentUtils.h"
 #include "nsFrameSelection.h"
 